README.md

```diff
@@ -13,13 +13,18 @@ Create app.cfg file from the app.template.cfg, which contains the Girder configu
 [girder]
 url: URL to Girder
 api_root: suffix to Girder API (usually api/v1)
+default_location: default girder location at app opening
 
 [ui]
 name: Application title to display in the toolbar
+
+[download]
+directory: path to folder to temporarily store downloaded files
+cache_mode: set how to handle the cache
```

Comment:
should be `=` instead of `:`

Reply:
instead of duplicating app.template.cfg, you should be able to "insert" the file here.